packages/cypress/src/generators/cypress-project/schema.json

```diff
@@ -59,6 +59,19 @@
       "type": "boolean",
       "default": false,
       "description": "Do not add dependencies to `package.json`."
+    },
+    "rootProject": {
+      "description": "Create a application at the root of the workspace",
+      "type": "boolean",
+      "default": false,
+      "hidden": true
+    },
+    "bundler": {
+      "description": "The Cypress builder to use.",
```

Comment:
property is called bundler, but the description says builder. not sure if that matters? 'Cypress builder' isn't really a term used in this context.